glide.lock

```diff
@@ -105,9 +105,9 @@ imports:
   subpackages:
   - conn
 - name: github.com/libvirt/libvirt-go
-  version: b8570d1869813105e8fb57b728900a98291658f7
+  version: d0e4392dfcc5489797a5f875cc3355b255612a18
```

Comment:
i just noticed this.

We don't have libvirt-go explicitly defined in glide.yaml, yet somehow it gets pulled in by something else as a dependency?!

We should consider defining libvirt-go in glide.yaml and locking it down to a specific release. Otherwise changes to 'make dep-update' are going always jump our libvirt-go version to the latest commit whether we want to do that or not.

Reply:
Will set it to `3.7` because we use this version under virt-launcher `Dockerfile`